x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/UnsupportedAttribute.java

```diff
@@ -163,17 +163,15 @@ public boolean hasCustomMessage() {
     }
 
     @Override
+    @SuppressWarnings("checkstyle:EqualsHashCode")// equals is implemented in parent. See innerEquals instead
     public int hashCode() {
-        return Objects.hash(super.hashCode(), hasCustomMessage, message);
+        return Objects.hash(super.hashCode(), message, hasCustomMessage);
```

Comment:
Why do we flip the order in the hash function?

Reply:
I regenerated equals/hashCode and they matched the field order.
Reverting to the prior behavior.